e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -159,6 +159,13 @@ describeIfIAMV2p1('projects API: applying project', () => {
     });
   });
 
+  beforeEach(() => {
+    cy.restoreStorage();
+  });
+  afterEach(() => {
+    cy.saveStorage();
+  });
```

Comment:
Is this already in use in other tests or is this something new and should be used in every test?

Reply:
This was needed when we were passing around the admin user's id token around. But after refactoring these tests to use the admin API token from the env, I took it out. 

I think all of the UI tests should be using these helpers - they help prevent the test getting kicked off by a session refresh. We mock the session/refresh response coming back with the same ID token each time - but if that ID token gets dropped btwn the tests because we're not saving storage, that mocked response will still lead to a forced logout.